content/ui-testing-handbook/react/en/user-flow-testing.md

```diff
@@ -62,12 +62,16 @@ Run: `yarn add -D cypress` to install the Cypress package. Then add the Cypress
 }
 ```
 
-Next, add a `cypress.json` file at the root of your project. Here we can configure the base URL for our application so that we don’t have to repeat ourselves when writing out actual test commands.
+Next, add a `cypress.config.js` file at the root of your project. Here we can configure the base URL for our application so that we don’t have to repeat ourselves when writing out actual test commands.
 
-```json:title=cypress.json
-{
-  "baseUrl": "http://localhost:3000"
-}
+```json:title=cypress.config.js
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: "http://localhost:3000/",
+  },
+});
```

Comment:
@timur-svoboda while following the tutorial and applying your suggestion, it seems that I've hit a bit of an issue. The suggestion you're proposing leads to the following when running `yarn cypress`:
```shell

Error: Your project does not contain a default supportFile. We expect a file matching cypress/support/e2e.{js,jsx,ts,tsx} to exist.

If a support file is not necessary for your project, set supportFile to false.

https://on.cypress.io/support-file-missing-or-invalid
    at setSupportFileAndFolder (/Users/myuser/Library/Caches/Cypress/10.11.0/Cypress.app/Contents/Resources/app/packages/config/src/project/utils.js:225:37)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at ProjectConfigManager.buildBaseFullConfig (/Users/myuser/Library/Caches/Cypress/10.11.0/Cypress.app/Contents/Resources/app/packages/data-context/src/data/ProjectConfigManager.js:351:26)
    at ProjectConfigManager.getFullInitialConfig (/Users/myuser/Library/Caches/Cypress/10.11.0/Cypress.app/Contents/Resources/app/packages/data-context/src/data/ProjectConfigManager.js:404:34)
    at ProjectConfigManager.setupNodeEvents (/Users/myuser/Library/Caches/Cypress/10.11.0/Cypress.app/Contents/Resources/app/packages/data-context/src/data/ProjectConfigManager.js:181:28)
```

To prevent this issue, I was required to add the following configuration element to `cypress.config.js`:

```js
//cypress.config.js

const { defineConfig } = require("cypress");
module.exports = defineConfig({
  e2e: {
    baseUrl: "http://localhost:3000/",
    supportFile: false, // 👈🏼 Configuration element to prevent issues with the test
  },
});
```

Adding this no longer shows the error, and I can run the test. 

If you could add it (minus my comment), I'd appreciate it.